spec/factories/requests.rb

```diff
@@ -1,3 +1,18 @@
+# == Schema Information
```

Comment:
This feels out of scope.

Reply:
This was added automatically when I ran `annotate`.

Reply:
This is fine -- I asked @pbteja1998 to run `annotate` :)